src/components/Inputs.js

```diff
@@ -137,16 +139,20 @@ export function useHdPathInput(props) {
   );
 }
 
-const kTicketValidators = [validateTicket, validateNotEmpty];
-const kTicketTransformers = [prependSig];
+const kTicketValidators = [
+  validateTicket,
+  validatePatpByteLength(8), // tickets are 8 bytes of patp
```

Comment:
This isn't strictly true. In development we may user shorter tickets, in production stars and galaxies usually have 16- and 32-byte tickets respectively.  
(Also tickets are patq, not patp, but more or less the same thing.)

Reply:
we probably want to allow 32 byte tickets, so `validateMaximumPatpByteLength(32)`?

I'll rename it to `validatePatpqByteLength` as well, since I guess it'll work for both encodings?

Reply:
We should just allow any valid patq, regardless of length. I wrote my bank an angry letter once about their needless limit on password length, Bridge definitely shouldn't make the same mistake. (^: